code/utils.py

```diff
@@ -615,6 +615,18 @@ def write_c1_grad_script(origsFile):
 
 # *********************************************************************************************************************
 
+def time_in_range(start, end, x):
```

Comment:
@kwhitehall @caocampb In iomethods.get_fileList_for_binaries function where utils.time_in_range is being called we don't have access to an instance of the UserVariables Class where the check_times method could be invoked.

After looking over the iomethods.get_fileList_for_binaries function we could replace this function with the following code:

if (start <= end) and (start <= dateAsDateTime <= end):    #this turns the function above into a one line conditional
    newFileList.append(file)

I haven't fully tested this code, but I think the logic should be sound.  The one benefit to having this function declared on it's own is we could (and should) write tests to double check the logic is complete.

@kwhitehall The one possible refactoring might be to take the check_times method and extract it and put it down in the utils module and have the UserVariables Class invoke it.  It would be a step toward keeping all time checks and validation in the utils module.